src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilerDiagnosticAnalyzer.CompilationAnalyzer.cs

```diff
@@ -116,7 +120,11 @@ public override int GetHashCode()
 
                 public override bool Equals(Diagnostic? obj)
                 {
-                    return _original.Equals(obj);
+                    if (obj instanceof CompilerDiagnostic cd) {
```

Comment:
```suggestion
                    if (obj is CompilerDiagnostic cd) {
```

Reply:
The change should also be following our coding style: braces on separate lines.